src/server/game/Quests/QuestDef.cpp

```diff
@@ -132,7 +132,15 @@ Quest::Quest(Field* questRecord)
 void Quest::LoadQuestDetails(Field* fields)
 {
     for (int i = 0; i < QUEST_EMOTE_COUNT; ++i)
+    {
+        if (fields[1+i].GetUInt16() >= EMOTE_MAX)
```

Comment:
Don't define EMOTE_MAX and don't check it like this
Verify the emote exists in Emotes.dbc instead (we load it, lookup sEmotesStore)